### ISSUE TITLE
Training loss Graph Issue

### ISSUE THREAD

**Issue by programme0419** [UMich-CURLY/Debias_IMU#4]
Hi, I’ve been studying your paper (https://arxiv.org/pdf/2504.09495) and found it very insightful.

I’m currently training a model based on your approach using my own UAV dataset. However, I’m observing that the training loss curve is quite noisy and shows large fluctuations across epochs. Interestingly, the validation error continues to decrease steadily as training progresses.

I was wondering if you’ve encountered similar behavior during your experiments, or if this might be related to characteristics of the dataset (e.g., noise, scale variation, or trajectory inconsistencies).

If possible, would you mind sharing your training loss curve per epoch? It would be very helpful for me to understand whether this behavior is expected or if I should further investigate issues in my dataset or training setup.

Thank you for your time, and I appreciate your work.

Best regards

<img width="532" height="246" alt="Image" src="https://github.com/user-attachments/assets/c4a1e027-0cfb-43a7-b77c-c5892199d64c" />

**Comment by Ben30230** [comment]
You can try the example code to obtain the training loss curve. I remember the training loss is continuously decreasing with some oscillation, like what you posed. This is indeed related to the characteristics of the dataset, but it will not impact the final validation/inference performance.
